processing/src/test/java/org/apache/druid/segment/data/NumericNullColumnSelectorTest.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.segment.data;
+
+import org.apache.druid.collections.bitmap.ImmutableBitmap;
+import org.apache.druid.collections.bitmap.WrappedRoaringBitmap;
+import org.apache.druid.segment.BaseNullableColumnValueSelector;
+import org.apache.druid.segment.ColumnValueSelector;
+import org.apache.druid.segment.SimpleAscendingOffset;
+import org.apache.druid.segment.column.DoublesColumn;
+import org.apache.druid.segment.column.FloatsColumn;
+import org.apache.druid.segment.column.LongsColumn;
+import org.apache.druid.segment.vector.NoFilterVectorOffset;
+import org.apache.druid.segment.vector.VectorValueSelector;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class NumericNullColumnSelectorTest
+{
+  private final int numRows = 1024;
+  private final int vectorSize = 128;
+  private final SimpleAscendingOffset offset = new SimpleAscendingOffset(numRows);
+  private final NoFilterVectorOffset vectorOffset = new NoFilterVectorOffset(vectorSize, 0, numRows);
+  private ImmutableBitmap bitmap;
+
+  @Before
+  public void setup()
+  {
+    WrappedRoaringBitmap mutable = new WrappedRoaringBitmap();
+    for (int i = 0; i < numRows; i++) {
+      if (ThreadLocalRandom.current().nextDouble(0.0, 1.0) > 0.7) {
```

Comment:
Maybe better to generate `N` random bitmaps with a fixed seed. That way, the test is deterministic (but still tests decent variety due to the `N` factor).